toolchain/sem_ir/file.h

```diff
@@ -70,6 +70,30 @@ class File : public Printable<File> {
     return types().GetAs<PointerType>(pointer_id).pointee_id;
   }
 
+  struct IntTypeInfo {
+    bool is_signed;
+    IntId bit_width = IntId::Invalid;
+  };
+
+  // Compute the core integer type information from a type ID.
+  //
+  // TODO: When we don't have a builtin int type mixed with actual `IntType`
+  // instructions, clients should directly query the `IntType` instruction to
+  // compute this information.
+  auto GetIntTypeInfo(TypeId int_type_id) const -> IntTypeInfo {
+    auto inst_id = types().GetInstId(int_type_id);
+    if (inst_id == InstId::BuiltinIntType) {
+      return {true, ints().LookupSigned(llvm::APInt(/*numBits=*/64, 32))};
```

Comment:
```suggestion
      return {true, ints().LookupSigned(llvm::APInt(/*numBits=*/64, /*val=*/32))};
```

I don't know what's what here without looking it up, and the "32" is also for bit width, so commenting both parameters would help me.

Alternately, you have a private constant for the 64-bit width on your store, had you considered making that public and used by callers? (mainly tests TBH, so I'm not sure this is really valuable) I think the name could instead get you to:

```suggestion
      return {true, ints().LookupSigned(llvm::APInt(IntStore::MinAPWidth, 32))};
```

Reply:
Simplified all of this by directly adding a `Lookup` that takes `int64_t` so we can use much simpler code. Does that work for you?